app/templates/suppliers/details.html

```diff
@@ -38,139 +37,191 @@
+                'href': url_for('.edit_what_buyers_will_see', _anchor = 'email'),
+                'text': 'Change',
+                'visuallyHiddenText': 'contact email'
+                }
+            ]
+        }
+    },
+
+    {
+      'key': {
+        'text': "Contact phone number"
+      },
+      'value': {
+        'text': supplier.contact.phoneNumber
+      },
+
+
+      'actions': {
+            'items': [{
+                'href': url_for('.edit_what_buyers_will_see', _anchor = 'phoneNumber'),
+                'text': 'Change',
+                'visuallyHiddenText': 'contact phone number'
+                }
+            ]
+        }
+    },
+
+    {
+      'key': {
+        'text': "Summary"
+      },
+      'value': {
+        'text': supplier.description
+      },
+      'actions': {
+            'items': [
+            {
+                'href': url_for('.edit_what_buyers_will_see', _anchor = 'description'),
+                'text': 'Change (optional)' if not supplier.description else 'Change',
+                'visuallyHiddenText': 'summary'
+                }]
+        }
+    },
+]
+}) }}
 
-{% endcall %}
+<h2 class="govuk-heading-m" id="registration_information">Company details for your framework applications</h2>
+{%  set has_address = supplier.contact.get("postcode") and supplier.contact.get("address1") and supplier.contact.get("city") and country_name %}
 
+{{ govukSummaryList({
+  'rows': [
+    {
+      'key': {
+        'text': "Registered company name"
+      },
+      'value': {
+        'text': supplier.registeredName
+      },
+      'actions': {
+          'items': [{
+            'href': url_for('.edit_supplier_registered_name'),
+            'text': 'Change' if supplier.registeredName else 'Answer required',
+            'visuallyHiddenText': 'registered company name' if supplier.registeredName else 'for registered company name'
+            }
+        ]
+      }
+    },
+
+    {
+      'key': {
+        'text': "Registered company address"
+      },
+      'value': {
+        'html': '<br />'.join([supplier.contact.get("address1"), supplier.contact.get("city"), supplier.contact.get("postcode"), country_name])
+      },
+
+      'actions': {
+            'items': [{
+                'href': url_for(".edit_registered_address"),
+                'text': 'Change' if has_address else 'Answer required',
+                'visuallyHiddenText': 'registered company address' if has_address else 'for registered company address'
+                }
+            ]
+        }
+    },
+
+    {
+      'key': {
+        'text': "Registration number"
+      },
+      'value': {
+        'text': company_details.get('registration_number')
+      },
+
+
+      'actions': {
+            'items': [{
+                'href': url_for(".edit_supplier_registration_number"),
+                'text': 'Change' if company_details.get('registration_number') else 'Answer required',
+                'visuallyHiddenText': 'registration number'
```

Comment:
Why does this not match the pattern of the other rows in this table?

Reply:
@bjgill 👀   
good spot, fixed in https://github.com/alphagov/digitalmarketplace-supplier-frontend/commit/057f6e8ec21965653fe46b993e9f5e25e99b7361